components/ModEditSubmissionForm.vue

```diff
@@ -620,34 +624,46 @@ const validateFields = () => {
-                    cityJa.value = submissionData['facility']?.contact?.address.cityJa || ''
-                    addressLine1Ja.value = submissionData['facility']?.contact?.address.addressLine1Ja || ''
-                    addressLine2Ja.value = submissionData['facility']?.contact?.address.addressLine2Ja || ''
-                    mapLatitude.value = submissionData['facility']?.mapLatitude?.toString() || ''
-                    mapLongitude.value = submissionData['facility']?.mapLongitude?.toString() || ''
+                    submissionFormFields.nameEn.value = submissionData['facility']?.nameEn || ''
+                    submissionFormFields.nameJa.value = submissionData['facility']?.nameJa || ''
+                    submissionFormFields.phone.value = submissionData['facility']?.contact?.phone || ''
+                    submissionFormFields.email.value = submissionData['facility']?.contact?.email || ''
+                    submissionFormFields.website.value = submissionData['facility']?.contact?.website || ''
+                    submissionFormFields.postalCode.value = submissionData['facility']?.contact?.address.postalCode || ''
+                    submissionFormFields.prefectureEn.value = submissionData['facility']?.contact?.address.prefectureEn || ''
+                    submissionFormFields.cityEn.value = submissionData['facility']?.contact?.address.cityEn || ''
+                    submissionFormFields.addressLine1En.value = submissionData['facility']?.contact?.address.addressLine1En || ''
+                    submissionFormFields.addressLine2En.value = submissionData['facility']?.contact?.address.addressLine2En || ''
+                    submissionFormFields.prefectureJa.value = submissionData['facility']?.contact?.address.prefectureJa || ''
+                    submissionFormFields.cityJa.value = submissionData['facility']?.contact?.address.cityJa || ''
+                    submissionFormFields.addressLine1Ja.value = submissionData['facility']?.contact?.address.addressLine1Ja || ''
+                    submissionFormFields.addressLine2Ja.value = submissionData['facility']?.contact?.address.addressLine2Ja || ''
+                    submissionFormFields.mapLatitude.value = submissionData['facility']?.mapLatitude?.toString() || ''
+                    submissionFormFields.mapLongitude.value = submissionData['facility']?.mapLongitude?.toString() || ''
                     break
                 case 'googleMapsUrl':
-                    googlemapsURL.value = submissionData['facility']?.contact?.googleMapsUrl || submissionData['googleMapsUrl']
+                    submissionFormFields.googlemapsURL.value = submissionData['facility']?.contact?.googleMapsUrl || submissionData['googleMapsUrl']
                     break
             }
         }
     }
+    for (const key in submissionFormFields) {
+        submissionBeforeChanges[key] = submissionFormFields[key].value
+    }
+}
```

Comment:
This function was updated to create a snapshot of the form before changes are made in `submissionBeforeChanges`.